bar.cpp

```diff
@@ -0,0 +1,18 @@
+#include "bar.hpp"
+
+#include <iostream>
+
+namespace test
+{
+    
+Bar::Bar(int x)
+{
+    m_x = x;
```

Comment:
Prefer initialization over assignment in constructors.

Reply:
fixed